about.html

```diff
@@ -0,0 +1,269 @@
+          </p>
+          <h1 class="mx-4 mt-2 display-5 primary-color text-center">
+            The Europe's leading tech festival!, TNW conference, NOW.
+          </h1>
+          <div
+            class="card m-4 mt-2 border-3 border-white bg-light"
+            id="intro-card"
+          >
+            <div class="card-body text-black-50">
+              TNW Conference is a 2-day technology event in the greater
+              Amsterdam area, where industry leaders and tech enthusiasts alike
+              come together, to explore how tech will shape the world of
+              tomorrow. <br /><br />
+              <ul class="only-desktop">
+                <li>
+                  We've invited 150+ speakers to share fresh and unique insights
+                  that our audience can bring back to the office
+                </li>
+                <li>
+                  There will be a 2,000 SQM expo floor, where companies can
+                  showcase their work, increase brand recognition, and generate
+                  new leads
+                </li>
+                <li>
+                  Our Startup Program helps 325+ participating startups gain a
+                  competitive advantage, and reach the next level through
+                  connections, business, and access to capital
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="text-center text-secondary w-75 mx-auto">
+            <p class="mt-5">
+              Build your brand, innovate your business, and drive action with
+              our audience.
+            </p>
+            <p class="contact">
+              <i>
+                Partner with us:
+                <a href="mailto:khiter.med7@gmail.com"
+                  ><u>tips@thenextweb.com.</u></a
+                ></i
+              >
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="logo-section">
+        <h4 class="mx-5 mt-4 display-6 secondary-color text-center">
+          The Europe's leading tech festival!, TNW conference, NOW.
+        </h4>
+        <div class="short-line mx-auto"></div>
+        <p class="text-black-50 mx-5 mt-4 text-center">
+          TNW Creative is here to help you build your brand and navigate the
+          world of tech. Our team of creatives will design your strategy around
+          your needs
+        </p>
+        <p class="only-desktop text-black-50 mx-5 mt-4 text-center">
+          TNW Talks are known for being daring, bold, inspirational, and
+          aspirational. Our 150+ expert speakers bring fresh new insights from
+          the tech industry that can help transform the world of tomorrow.
+        </p>
+        <div class="card m-5 mt-2 border-3 border-white bg-light">
+          <div class="card-body mx-auto">
+            <img
+              src="img/The_Next_Web_Logo.png"
+              alt="The_Next_Web_Logo"
+              class="logo"
+            />
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this specific card section of the mobile About Page is well aligned and centered in order to improve the UI of your design.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-23 00-28-30](https://user-images.githubusercontent.com/58692590/164817157-b25dc21e-3adb-4210-8ebe-3810d22af763.png) | ![Screenshot from 2022-04-23 00-21-28](https://user-images.githubusercontent.com/58692590/164817167-7bbdac2e-119a-43a4-a3db-45e2171880a7.png)

Reply:
Hi @Whoistolu, On my Browser and on any logic size It is well centered, you just opened the developer tool with a too low width that's why.